pkgs/applications/blockchains/wasabibackend/deps.nix

```diff
@@ -1,952 +1,955 @@
 { fetchurl }:
 let
   nugetUrlBase = "https://www.nuget.org/api/v2/package";
-  fetchNuGet = { name, version, sha256 }: fetchurl { inherit sha256; url = "${nugetUrlBase}/${name}/${version}"; };
-in
-[
+  fetchNuGet = { pname, version, sha256 }:
+    fetchurl {
+      inherit sha256;
+      url = "${nugetUrlBase}/${pname}/${version}";
+    };
+in [
```

Comment:
Can we use the top level function here, too?

Reply:
fine this way?